### ISSUE TITLE
New Account is not getting added & issue in updating existing account balance

### ISSUE THREAD

**Issue by ToufeeqP** [polkascan/polkascan-pre-harvester#16]
I have made some changes to the blockchain such that instead of storing AccountStore in System module, I am storing it in custom module & implemented all required Trait in that pallet & everything working fine. But to use polkascan with this customised blockchain, I have updated the polkascan code in a way that I have replaced System module for balances to Custom module. And `data_account_info_snapshot` table is getting updated correctly but the new account entry is not getting created `data_account`. Any suggestion or help will highly be appreciated.

To summarise, I have customer balance pallet which binds balance to Did[u8; 32] instead of AccountId & AccountStore for this custom balance pallet is another custom pallet instead of System module. And Accounts Balances or not getting updated for genesis Accounts & not getting created new accounts (Endowed).
